intelmq/bin/intelmqctl.py

```diff
@@ -767,8 +767,9 @@ def read_bot_log(self, bot_id, log_level, number_of_lines):
             bot_log_path = os.path.join(self._parameters.logging_path,
                                         bot_id + '.log')
             if not os.path.isfile(bot_log_path):
-                self._logger.error("Log path not found: %s", bot_log_path)
-                return 2, []
+                message = {'date': datetime.datetime.now().isoformat(), "bot_id": bot_id, "log_level": "INFO", "message": 'No log file exists (yet)'}
```

Comment:
```suggestion
                message = {'date': datetime.datetime.now().isoformat(), "bot_id": bot_id, "log_level": "INFO", "message": 'No log file exists (yet).'}
```
we have dots at the end of log messages nearly everywhere